runtime/src/consensus/tendermint/merkle.rs

```diff
@@ -0,0 +1,173 @@
+//! Merkle proofs used in Tendermint networks
+//!
+//! Rewritten to Rust from:
+//! https://github.com/tendermint/tendermint/blob/main/crypto/merkle/proof.go
```

Comment:
So there is no Rust implementation of this scheme yet?

Reply:
It seems not. Was searching without any success.